src/component/modal/addressModal.css

```diff
@@ -0,0 +1,32 @@
+.address-modal{
+    text-align: center;
+    position: fixed;
+    top:50%;
+    left:50%;
+    transform: translate(-50%,-50%);
+    background-color: var(--bg-secondary);
+    width: 400px;
```

Comment:
Suggestion - Using rem here for bigger px values will help in responsiveness

Reply:
yep, but I am redefining it for smaller devices